samples/robotframework-browser-library/tests/homepage-tests.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Library   Browser
+
+*** Test Cases ***
+Website Launch Test And Verify Presence Of Header Links
```

Comment:
Can we call this simply `Verify Presence Of Header Links`?

Reply:
done